app/models/wishlist.rb

```diff
@@ -0,0 +1,4 @@
+class Wishlist < ApplicationRecord
+  validates :name, length: { in: 3..30 }
+  validates :description, length: { in: 10..150 }
```

Comment:
If we want to restrict lengths of those strings, then we'd better ask @Schocker and @agalkiewicz and other front-devs that have some ux ideas. Maybe it'd be a better user experience to actually limit the description to let's say 140 chars. My point is, **if we're going to limit the length then the chosen value, there should be valid a reason behind it**, and not some random number from the top of our heads. Right now 500 is as good as any, let's just get this PR to be merged, and move on. We'll change it when needed.